src/fetch_uri.c

```diff
@@ -163,6 +163,31 @@ archive_finish_callback (gpointer user_data)
     g_slice_free(FetchData, fetch_data);
     return FALSE;
 }
+static gboolean
+is_entry_prefixed_with_fragment(const gchar *entry_path,
+                                const gchar *fragment)
+{
+   
+    gchar *skip_entry_branch = NULL;
+   
+    if (fragment == NULL)
+        return FALSE;
+
+    skip_entry_branch = g_strstr_len(entry_path, -1, "/");
```

Comment:
Shouldn't this be searching from the left side of the string?  If searching from the right side of the string it will stop at the first / right?

Reply:
oh... nevermind..  I can't read Today. :-)  I thought it was g_strrstr_len.

Reply:
I too fell in that trap when I was reviewing choices in glib doc.